src/compiler/checker.ts

```diff
@@ -26774,6 +26781,12 @@ namespace ts {
             diagnostics.add(resultDiagnostic);
         }
 
+        function containerSeemstoBeEmptyDomElement(containingType: Type) {
+            return (compilerOptions.lib && !compilerOptions.lib.includes("dom")) &&
+                everyContainedType(containingType, type => /^(EventTarget|Node|((HTML[a-zA-Z]*)?Element))$/.test(type.symbol.escapedName.toString())) &&

```

Comment:
@sandersn was there a reason you asked to not use a regexp in https://github.com/microsoft/TypeScript/pull/34407#discussion_r389982833? HTML*Element seems easiest to check that way.

I'm also not clear on whether `isEmptyObjectType` or `getPropertiesOfType` approach here would be preferred. Do you have guidance?